examples/fig_setup.py

```diff
@@ -0,0 +1,324 @@
+#!/usr/bin/env python2
+# encoding: utf-8
+"""Example script for a plot using the gridspeccer module.
+
+Neccessary functions are get_gridspec, adjust_axes, plot_labels and get_fig_kwargs.
+The actual plot functions are below and are named in accordance to the names in
+the get_gridspec function.
+"""
+
+from matplotlib import gridspec as gs
+import numpy as np
+import matplotlib.patches as mpatches
+import matplotlib.pyplot as plt
+
+from gridspeccer import core
+from gridspeccer.core import log
+from gridspeccer import aux
+
+
+def get_gridspec():
+    """
+        Return dict: plot -> gridspec
+    """
+
+    gs_main = gs.GridSpec(1, 1,
+                          left=0.05, right=0.97, top=0.95, bottom=0.08)
+
+    gs_schematics = gs.GridSpecFromSubplotSpec(2, 1, gs_main[0, 0],
+                                               height_ratios=[1.5, 2.0],
+                                               hspace=0.2)
+    gs_tikz = gs.GridSpecFromSubplotSpec(1, 2, gs_schematics[1, 0],
+                                         width_ratios=[2, 1])
+    gs_lower_row = gs.GridSpecFromSubplotSpec(1, 2, gs_schematics[0, 0],
+                                              wspace=0.35,
+                                              width_ratios=[1.5, 1])
+    gs_membranes = gs.GridSpecFromSubplotSpec(2, 1, gs_lower_row[0, 1],
+                                              hspace=0.2)
+
+    return {
+        # ### schematics
+        "arch": gs_tikz[0, 0],
+
+        "coding": gs_tikz[0, 1],
+
+        "psp_shapes": gs_lower_row[0, 0],
+
+        "membrane_schematic_0": gs_membranes[0, 0],
+        "membrane_schematic_1": gs_membranes[1, 0],
+    }
+
+
+def adjust_axes(axes):
+    """
+        Settings for all plots.
+    """
+    for axis in axes.values():
+        core.hide_axis(axis)
+
+    for k in [
+        "arch",
+    ]:
```

Comment:
Should probably be an explicitly named variable `axes_frame_disabled`.
Not really important for now, though.